site/themes/carvel/layouts/partials/_footer.html

```diff
@@ -8,11 +8,18 @@
 			</ul>
 			<div class="right-links">
 				<p>
-					<a target="_blank" rel="noopener" href="/"> <img src="/img/logo.svg" /></a>
+					<a target="_blank" rel="noopener" href="/"> <img src="/img/logo.svg" /></a>	
 				</p>
+				
 			</div>
 		</div>
 		<div class="bottom-links">
+			<p> 
+				We are a Cloud Native Computing Foundation sandbox project. 
+				<a target="_blank" rel="noopener" href="https://www.cncf.io/">
+					<img id="cncf-logo" src="/img/cncf-color.svg" alt="CNCF logo" />
+				</a>
+			</p>
```

Comment:
there is text below "A VMware-backed project."

Reply:
yeh this is a somewhat hacked together attempt at web development. I must admit that I haven't ventured into this realm in ... ever? lol

This is just a draft, will update it accordingly when I know its correct